MyBus/MyBus/ViewController.swift

```diff
@@ -214,15 +207,23 @@ class ViewController: UIViewController, UIPopoverPresentationControllerDelegate,
         }
     }
 
-    func getMarkerImage(imageResourceIdentifier : String, annotationTitle : String) -> MGLAnnotationImage {
+    func mapViewRegionIsChanging(mapView: MGLMapView) {
+        mapView.showsUserLocation = false
+    }
+
+    func mapView(mapView: MGLMapView, didFailToLocateUserWithError error: NSError) {
+        print("error locating user")
```

Comment:
I would add more information about the error with `print("error locating user: \(error.localizedDescription)")`